src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -251,7 +260,49 @@ bool ControllerScriptEngineLegacy::callInitFunction() {
             // connection QQmlEngine::warnings ->
             // ControllerScriptEngineBase::handleQMLErrors
         }
-        return success;
+
+        QListIterator<std::shared_ptr<mixxx::qml::MixxxController>> controllers(m_mixxxController);
+        bool controllersSuccess = true;
+        while (controllers.hasNext()) {
```

Comment:
Please use a for-loop instead

```suggestion
        bool controllersSuccess = true;
        for (auto controller: m_mixxxController) {
```